app/src/main/java/com/example/hm5_coroutinesflow/adapter/PersonViewHolder.kt

```diff
@@ -0,0 +1,39 @@
+package com.example.hm5_coroutinesflow.adapter
+
+import android.content.ClipData
+import android.widget.Toast
+import androidx.recyclerview.widget.RecyclerView
+import coil.load
+import coil.size.Scale
+import coil.size.ViewSizeResolver
+import com.example.hm5_coroutinesflow.databinding.ItemPersonBinding
+import com.example.hm5_coroutinesflow.model.CartoonPerson
+import com.example.hm5_coroutinesflow.model.ItemType
+
+
+// сюда нужно передать вторым параметром еще инфу по фото или АПИ  https://youtu.be/IDVxFjLeecA?t=10544
+class PersonViewHolder(
+    private val binding: ItemPersonBinding,
+    private val onUserClicked: (ItemType<CartoonPerson>) -> Unit,
```

Comment:
здесь ты работаешь с определенным типом элемента, то есть контент, поэтому тебе нужно в лямбде и методе bind работать только с теми элементами, которые предполагаются
```suggestion
    private val onUserClicked: (CartoonPerson) -> Unit,
```